crates/cargo-remove/src/cargo/ops/cargo_remove/mod.rs

```diff
@@ -75,6 +79,7 @@ pub fn remove(options: &RemoveOptions<'_>) -> CargoResult<()> {
             .warn("aborting remove due to dry run")?;
     } else {
         manifest.write()?;
+        resolve_ws(options.workspace)?;
```

Comment:
How come this wasn't done in the `bin` like cargo-add?

I believe my intention with add was from an API perspective: the caller of this might want to do other operations before updating the workspace

Reply:
Ah, I was wondering. That makes sense.